doc/source/cost.rst

```diff
@@ -38,7 +38,7 @@ you.
 
 .. raw:: html
 
-   <a target="_blank" href="http://beta.mybinder.org/v2/gh/jupyterhub/zero-to-jupyterhub-k8s/master?filepath=draw_function.ipynb">
+   <a target="_blank" href="http://beta.mybinder.org/v2/gh/jupyterhub/zero-to-jupyterhub-k8s/move_ntbk?filepath=doc/ntbk/draw_function.ipynb">
```

Comment:
Is this a branch it's pointing to? If so you should replace that with either a commit hash or a tag :)

Reply:
that was actually just to double-check that this PR branch works...I'm gonna switch it to a tag w/ the next push!